### DISCUSSION TITLE
Unable to upgrade ArgoCD from 2.11.0 to 2.11.x.

### DISCUSSION THREAD

**Discussion by mdinhvan** [argoproj/argo-cd#18534] [Q&A]
Hi Everyone,
Can you guys assist with the below error while trying to upgrade ArgoCD from 2.11.0 to 2.11.x (2.11.1, 2.11.2, 2.11.3)

![image](https://github.com/argoproj/argo-cd/assets/89178125/53c79d3a-188d-4e87-bb2f-32d61723f8e8)

It seems like related to Redis changes.

Thanks.

**Comment by crenshaw-dev** [comment]
Are you using the official Helm chart? If so, I'd recommend opening an issue on the argo-helm repo.
